wallet/src/main/AndroidManifest.xml

```diff
@@ -54,6 +56,7 @@
 
 		<activity
```

Comment:
Could also add `exported="false"`, since the onboarding activity is only referenced internally